bots/service.go

```diff
@@ -152,18 +182,19 @@ func (s *Service) Create(botID string, botVersion int, name string, params map[s
 		return
 	}
 
-	store := &BotKVStore{
-		botID,
-		botVersion,
+	store := Datastore{
+		ds.NewKey(botID),
```

Comment:
if it was a generic implementation i'd go for it, but since this is the default fulfillment of the kv it seems like it's not required (since the table is the "bots" namespace)